src/placeholder.js

```diff
@@ -18,8 +18,22 @@ class LiveCardModal extends ModalDialog {
    */
   constructor(player, options) {
     super(player, options);
-    this.on(player, 'error', this.open);
+    this.on(player, 'error', function () {
+      //display error card if no stream found 
+      var errNo = player.error().code;
+      var duration = player.duration();
+      console.log(duration);
```

Comment:
Done, thanks ben

On 17 June 2016 at 15:02, mister-ben <notifications@github.com> wrote:

> In src/placeholder.js
> <https://github.com/mister-ben/videojs-live-card/pull/1#discussion_r67514619>
> :
>
> > @@ -18,8 +18,22 @@ class LiveCardModal extends ModalDialog {
> >     */
> >    constructor(player, options) {
> >      super(player, options);
> > -    this.on(player, 'error', this.open);
> > +    this.on(player, 'error', function () {
> > +      //display error card if no stream found
> > +      var errNo = player.error().code;
> > +      var duration = player.duration();
> > +      console.log(duration);
>
> Could you remove the console.log?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mister-ben/videojs-live-card/pull/1/files/ab69aa41aebee86730acbda6b3121e039f4f94eb#r67514619>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJlTUjJJ8q3Qc0me5zuli4mFiBpXmjs6ks5qMqjngaJpZM4IX6IP>
> .
>